arm/Microsoft.ContainerRegistry/registries/readme.md

```diff
@@ -41,6 +41,71 @@ Azure Container Registry is a managed, private Docker registry service based on
 | `trustPolicyStatus` | string |  |  | Optional. The value that indicates whether the policy is enabled or not. |
 | `workspaceId` | string |  |  | Optional. Resource ID of log analytics. |
 
+### Parameter Usage: `privateEndpoints`
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done